addons/im_livechat/views/mail_channel_views.xml

```diff
@@ -84,6 +84,11 @@
             <field name="search_view_id" ref="im_livechat.mail_channel_view_search"/>
             <field name="domain">[('livechat_channel_id', '!=', None)]</field>
             <field name="context">{}</field>
+            <field name="help" type="html">
+                <p>
+                    Your chatter history is empty. Create a Channel and start chatting to fill up your history.
```

Comment:
Why is there an uppercase C on the word "channel" ? Please change it. Thank you.